app/views/transactions/destroy.js.slim

```diff
@@ -0,0 +1 @@
+| $('.delete-tr').bind('ajax:success', function() { $(this).closest('tr').fadeOut(); });
```

Comment:
@marderer why didn't you use webpack pipeline? change base brunch if needed

Reply:
@AleksSenkou Maybe you can recommend me a good article or a video about it? And can I do this later in another PR?